File: api-tester/src/main/java/com/revenuecat/apitester/java/OfferingsAPI.java
package com.revenuecat.apitester.java;

import com.revenuecat.purchases.Offering;
import com.revenuecat.purchases.Offerings;

import java.util.Map;

@SuppressWarnings({"unused"})
final class OfferingsAPI {
    static void check(final Offerings offerings) {
        final Offering current = offerings.getCurrent();
        final Map<String, Offering> all = offerings.getAll();
        final Offering o1 = offerings.getOffering("");
        final Offering o2 = offerings.get("");
        final Offering o3 = offerings.getCurrentOfferingForPlacement("")
    }
}

File: api-tester/src/main/java/com/revenuecat/apitester/kotlin/OfferingsAPI.kt
package com.revenuecat.apitester.kotlin

import com.revenuecat.purchases.Offering
import com.revenuecat.purchases.Offerings

@Suppress("unused", "UNUSED_VARIABLE")
private class OfferingsAPI {
    fun check(offerings: Offerings) {
        with(offerings) {
            val current: Offering? = current
            val all: Map<String, Offering> = all
            val o1: Offering? = getOffering("")
            val o2: Offering? = this[""]
            val o3: Offering? = getCurrentOfferingForPlacement("")
        }
    }
}
